### DISCUSSION TITLE
Number of available models decreased from useeior 1.1.1 to 1.5.1

### DISCUSSION THREAD

**Discussion by fchichorro** [USEPA/useeior#319] [Q&A] [answered]
useeior@1.1.1 seems to have many more available models than useeior@1.5.1. Is this intended behaviour?

```R
# Install useeior
remotes::install_github("USEPA/useeior@v1.1.1")
#load useeior
library("useeior")
seeAvailableModels()
 [1] "GAEEIOv2.0-s-WAT"            "USEEIOv2.0-411"              "USEEIOv2.0-79-GHG"           "USEEIOv2.0-GHG-NGCombustion" "USEEIOv2.0-GHG"              "USEEIOv2.0-i-411"           
 [7] "USEEIOv2.0-i-GHG"            "USEEIOv2.0-is-GHG"           "USEEIOv2.0-s-GHG"            "USEEIOv2.0.1-411"            "USEEIOv2.0"                  "USEEIOv2.1-422" 

# Install useeior (latest)
remotes::install_github("USEPA/useeior@v1.5.1")
#load useeior
library("useeior")
seeAvailableModels()
[1] "USEEIOv2.0.1-411"
```

**Comment by bl-young** [comment, marked as answer]
Hi @fchichorro  - good spot.

Rather than store many iterations of models within the package, many model specs have been moved. Including to some other repositories, e.g. [USEEIO-State](https://github.com/USEPA/USEEIO-State/tree/main/model_specs).

We also now test a number of advanced features using model specs hosted in the tests subfolder, [here](https://github.com/USEPA/useeior/tree/master/tests/modelspecs). Because these are not officially published models but rather intended to test iterations of model building we have pulled them out of the core model_specs folder.

You can find links to some other model specs here: https://www.epa.gov/land-research/us-environmentally-extended-input-output-useeio-technical-content
